packages/shared/src/lib/featureManagement.ts

```diff
@@ -137,6 +139,24 @@ export class Features<T extends FeatureValue = string> {
     [InAppNotificationPositionEnum.Bottom, InAppNotificationPositionEnum.Top],
   );
 
+  static readonly OnboardingV2 = new Features(
+    'onboarding_v2',
+    OnboardingV2.Control,
+    [OnboardingV2.Control, OnboardingV2.V1],
+  );
```

Comment:
Non breaking, but reading these broke my head a bit 😂
OnboardingV2.V1 seems very counter intuative, but for now let's keep it.
Just a headsup for future we might need to think a bit better on naming features.

Reply:
Yeah, we will definitely need to come up with something more understandable 😂